README.md

```diff
@@ -55,6 +55,7 @@ func main() {
     r.HandleFunc("/products", ProductsHandler)
     r.HandleFunc("/articles", ArticlesHandler)
     http.Handle("/", r)
```

Comment:
Replace both these lines with:

```go
log.Fatal(http.ListenAndServe(":8080", r))`
```

I would also add a comment above the handlers, along the lines of:

```go
// Each HandleFunc takes a path to match and a http.HandlerFunc to call on that match
```